### PR TITLE
Add tests

### LINKED ISSUE #1: Unit tests missed ! Help wanted !
Looking for someone who wants to contribute writing unit tests using react-testing-library!

### PR DESCRIPTION
**Pull request by sbardian**
Closes #1

### CODE REVIEW

File: __tests__/index.spec.js [hunk 1]
```diff
@@ -1,7 +1,59 @@
-// import useInputDebounce from "./../src";
+import React from "react";
+import { render, fireEvent, cleanup } from "react-testing-library";
+import useInputDebounce from "./../src";
+import { debug } from "util";
```
Comment (iusehooks) [inline __tests__/index.spec.js]: Would you remove this import? you are not using it anywhere.

File: __tests__/index.spec.js [hunk 2]
```diff
@@ -1,7 +1,59 @@
...
+  });
+  it("called with no delay or minLength", () => {
+    const { getByTestId } = render(<DebouncedComponent />);
+    const dInput = getByTestId("my-test-input");
+    fireEvent.change(dInput, { target: { value: "ab" } });
+    jest.runAllTimers();
+    expect(effect).toBeCalledTimes(1);
+  });
+  it("setTimeout", () => {
+    const { getByTestId, debug } = render(
```
Comment (iusehooks) [inline __tests__/index.spec.js]: would you remove it even from here 😀?

**Review by iusehooks — CHANGES_REQUESTED** [review]

File: __tests__/index.spec.js (hunk 1 above)
Comment (sbardian) [reply __tests__/index.spec.js]: removed.

File: __tests__/index.spec.js (hunk 2 above)
Comment (sbardian) [reply __tests__/index.spec.js]: removed.

**Comment by iusehooks** [comment]
Thanks for your contribution, appreciated indeed.

**Comment by sbardian** [comment]
> Thanks for your contribution, appreciated indeed.

No problem.  New to hooks testing so was good practice. 👍🏼

**Pull request merged**